.github/workflows/ci.yml

```diff
@@ -2,8 +2,6 @@ name: CI
 
 on:
   push:
-    branches:
-      - main
     tags:
       - v[0-9]+.[0-9]+.[0-9]+*
```

Comment:
Is there a use case where we need it to run on tags as well? Seems like that would just lead to a redundant run when `semantic-release` pushes the tag.

Reply:
https://github.com/electron-userland/electron-installer-common/pull/218/commits/bbce8bd36de5a696db8cb6450b7303a121eb07bc